### PR TITLE
Add recipe for 3deyes's Tesseract Translator

### PR DESCRIPTION
**Pull request by phoudoin**

### CODE REVIEW

**Comment by phoudoin** [comment]
See #1487

File: haiku-apps/tesseracttranslator/tesseracttranslator-0.1.recipe [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	lib:libtesseract$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libtesseract$secondaryArchSuffix
+	devel:liblept$secondaryArchSuffix
+	"
+#	tesseract${secondaryArchSuffix}_devel
+#	leptonica${secondaryArchSuffix}_devel
```
Comment (waddlesplash) [inline haiku-apps/tesseracttranslator/tesseracttranslator-0.1.recipe]: You can remove these comments.

File: haiku-apps/tesseracttranslator/tesseracttranslator-0.1.recipe [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc
```
Comment (waddlesplash) [inline haiku-apps/tesseracttranslator/tesseracttranslator-0.1.recipe]: This needs to have a $secondaryArchSuffix, and there also needs to be a `cmd:ld$secondaryArchSuffix`.

File: haiku-apps/tesseracttranslator/tesseracttranslator-0.1.recipe (hunk 1 above)
Comment (phoudoin) [inline haiku-apps/tesseracttranslator/tesseracttranslator-0.1.recipe]: Oups.

**Comment by phoudoin** [comment]
PR fixed

**Comment by korli** [comment]
The SOURCE_URI is at least incorrect:
SOURCE_URI="http://github.com/threedeyes/TesseractTranslator/archive/master.zip"
Should be a commit.

**Comment by phoudoin** [comment]
I fail to see what's wrong witht the current commited SOURCE_URI.

**Comment by scottmc** [comment]
instead of master.zip use: https://github.com/threedeyes/TesseractTranslator/archive/0107c23aac00ffd5a331372d9f26bc2c102981a3.zip
As this points to a specific commit, rather than the current master.

**Comment by phoudoin** [comment]
Oh. Okay, will fix that.

**Pull request merged**